quickstart/armv7a9-zynq7000-zturn.md

```diff
@@ -0,0 +1,245 @@
+# Running system on `armv7a9-zynq7000-zturn` (Xilinx Zynq-7000, Zturn development board)
+
+These instructions describe how to run a Phoenix-RTOS system image for `armv7a9-zynq7000-zturn` target architecture.
+Note that, the build artifacts, including the system image, should be first provided in the `_boot` directory.
+If you haven't run the `build.sh` script yet, run it for `armv7a9-zynq7000-zturn` target.
+
+See [how to build the Phoenix-RTOS system image](../building/README.md).
+
+## Preparing the board
+
+Preparing the board depends on how the plo is loaded into RAM, this quickstart describes 2 approaches - loading from SD
+card and NAND flash, depending on your needs use one of them. For example if you have empty flash memory or want to
+load new plo use SD card, otherwise you can simply load plo from NAND flash.
+
+### Load plo from SD card
+
+- Firstly, prepare SD card, I would prefer to format it to FAT32
+
+- Secondly, you should copy the disk image `phoenix.disk` from the `_boot/armv7a9-zynq7000-zturn` directory to the SD
+  card and rename it to `BOOT.bin`, make sure that this file is in binary format, otherwise you won't be able to load
+  plo (Phoenix-RTOS Loader) from SD card to RAM.
+
+- Then, insert the SD card into the board.
+
+- To allow loading from SD card, set the jumpers to the following configuration:
+
+  <img src="_images/zynq7000-zturn-sd-boot.jpg" width="400px">
+
+### Load plo from NAND flash
+
+- To allow load from NAND flash, set the jumpers to the following configuration:
+
+  <img src="_images/zynq7000-zturn-flash-boot.jpg" width="400px">
+
+### Next steps are common for both approaches
+
+- To communicate with the board you will need to connect the USB cable to the `USB_UART` port (`J6`).
+
+- You should also connect another micro USB cable to the `USB_OTG` port (`J2`).
```

Comment:
Please add a note about jumper that changes power source.

```suggestion
- To power the board you should also connect another micro USB cable to the `USB_OTG` port (`J2`). Make sure that JX jumper is in xxx position!
```

Reply:
There is no such jumper